### ISSUE TITLE
RinkWorks: Book-a-minute-classics

### ISSUE THREAD

**Issue by WhitneyGomes** [nikolaswise/web-design-workshop#69]
RinkWorks is an "entertainment" web site based in New Hampshire that compiles––and therefore is part of––the useless web. 
<!DOCTYPE html>
<html>
<body>
<a href="http://www.rinkworks.com/bookaminute/classics.shtml"> Book-a-minute-classics</a>
is the epitome of an oxymoron: an instant classic. A classically really, really bad website. 
<img width="651" alt="screen shot 2017-05-06 at 12 17 27 am" src="https://cloud.githubusercontent.com/assets/27308999/25770703/76448de2-31f1-11e7-8e74-8e52cd2168d5.png">

</body>
</html>

- The only two options for site navigation at the header of the page are "main" and "site guide," which will link you to the RinkWorks page, in which "Book-a-minute-classics" is not even featured.

- This site was clearly built using very basic html, which is fine, but the promise to offer students a comprehensive cheat guide to book reports is a trap. Looking at the quality of the site, any high school student depending on this website's scheme deserves to get an "F".

- Poor communication: information is not organized (no headlines, subheads, bullet points, or sections to denote new pages of content). The book titles are not alphabetized and they are arbitrarily placed one after another on the home page. There is no logic in their order.

- Font: I appreciated the old-school Times New Roman, but it is far too small for users to read anything on the screen––especially if they have impaired vision. 

In short, it's a terrible website, **BUT** the point of the website is to entertain. Please do yourself a favor and click on some of the book "reports". They are pretty hilarious.
